modules/cas_cache/layer_cache_management.c

```diff
@@ -2188,6 +2188,19 @@ int cache_mngt_get_cache_params(struct kcas_get_cache_param *info)
 				ocf_cleaning_acp, ocf_acp_flush_max_buffers,
 				&info->param_value);
 		break;
+	case cache_param_promotion_policy_type:
+		result = cache_mngt_get_promotion_policy(cache, &info->param_value);
+		break;
+	case cache_param_promotion_nhit_insertion_threshold:
+		result = cache_mngt_get_promotion_param(cache,
+				ocf_nhit_insertion_threshold,
+				(uint64_t *)&info->param_value);
+		break;
+	case cache_param_promotion_nhit_trigger_threshold:
+		result = cache_mngt_get_promotion_param(cache,
+				ocf_nhit_trigger_threshold,
+				(uint64_t *)&info->param_value);
```

Comment:
This looks dangerous. Please change type in OCF interface to 32 bit and remove this cast here.

Reply:
Done